### PR TITLE
Add support fo Maya 2023 plugin

### PR DESCRIPTION
**Pull request by pandaGaume**
Initial version of Maya2023 exporter plugin.
Nothing seems changed so far into the SDK Api, wee may conduct more tests with the help of the comunity.

### CODE REVIEW

File: Maya/Maya2Babylon.sln [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
 Microsoft Visual Studio Solution File, Format Version 12.00
-<<<<<<< Updated upstream
 
-# Visual Studio 15
-VisualStudioVersion = 15.0.26730.12
-
-=======
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.31613.86
->>>>>>> Stashed changes
```
Comment (bghgary) [inline Maya/Maya2Babylon.sln:11]: lol

File: Maya/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -6,11 +6,11 @@
 // l'ensemble d'attributs suivant. Changez les valeurs de ces attributs pour modifier les informations
 // associées à un assembly.
 [assembly: AssemblyTitle("Maya2Babylon")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("Maya to Babylon exporter plugin.")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
+[assembly: AssemblyCompany("Microsoft")]
```
Comment (bghgary) [inline Maya/Properties/AssemblyInfo.cs:11]:
```suggestion
[assembly: AssemblyCompany("Microsoft Corporation")]
```

**Review by bghgary — APPROVED** [review]

**Comment by bghgary** [comment]
I guess we need to add Maya 2023 to the bulid?

**Comment by 0k1019** [comment]
Nice to support for Maya 2023 plugin. I try to contribute about build script, but I can't.
Support build Maya 2023 plugin via release later.

**Comment by bghgary** [comment]
I updated the build to include Maya 2023. Not sure I did it right yet, but hopefully it will work soon.

**Comment by bghgary** [comment]
It's out: https://github.com/BabylonJS/Exporters/releases/tag/Release20220613.1

Can someone try it to make sure it's working?

**Comment by pandaGaume** [comment]
Thank's, i just tried and it's working. I have to setting up the installer now. Then new PR will follow soon.

**Comment by 0k1019** [comment]
Thank you. It works to me.

**Pull request merged**